### DISCUSSION TITLE
Customize configuration files through configmaps (routr-connect Helm chart)

### DISCUSSION THREAD

**Discussion by alvitoraidhy** [fonoster/routr#234] [Ideas]
**Is your feature request related to a problem? Please describe.**
We're looking to deploy this as a subchart with another one containing Asterisk instances. Ideally this would be pre-configured as in once deployed Routr would loadbalance the asterisk instances automatically. However as far as I understand we need to add peers through the CLI first after deploying.

**Describe the solution you'd like**
If we could have parameters to define peers, credentials, etc I think they would be great additions to the chart.

**Describe alternatives you've considered**
-

**Additional context**
Configuration files: https://github.com/fonoster/routr/tree/3897ad320d98a0d17ca7dd552fc0ca7c5a84368d/config/resources

**Comment by psanders** [comment]
I'm converting this into a conversation. Although I believe we can improve the process of seeding the database, I don't think it belongs in the Helm chart. Perhaps we could introduce a command for rctl to seed the DB from a JSON file?
